### PR TITLE
feat(cijoe/bench): add latency barplot for SPDK report

### PR DESCRIPTION
**Pull request by NaddiNadja**
There is no prose introducing the barplot. This will be added later.

### CODE REVIEW

File: cijoe/scripts/bench_plotter.py [hunk 1]
```diff
@@ -83,22 +82,34 @@ def plot_attributes_from_step(step):
     }
 
 
-def draw_bar_plot(data, plot_attributes, xlabel=None, y_limit=None):
+def draw_bar_plot(data, plot_attributes, xlabel=None, ylabel="iops", y_limit=None):
```
Comment (karlowich) [inline cijoe/scripts/bench_plotter.py:85]: I think we should decide on a convention for the x/y-labels. 
Either they should both include a default string or both default to `None`.

File: cijoe/scripts/bench_plotter.py [hunk 2]
```diff
@@ -83,22 +82,34 @@ def plot_attributes_from_step(step):
...
 
     groups = list(set(map(lambda item: item["group"], data.values())))
     is_grouped = len(groups) > 1
 
-    num_bars = len(data.items()) - 1
     if is_grouped:
         x_ticks = dict()
-        x_tick_info = dict([(group, [groups.index(group), 0]) for group in groups])
+
+        # fiddly stuff to get groups of uneven sizes to have even space betweem them
```
Comment (karlowich) [inline cijoe/scripts/bench_plotter.py:96]:
```suggestion
        # fiddly stuff to get groups of uneven sizes to have even space between them
```

File: cijoe/scripts/latency_plotter.py [hunk 3]
```diff
@@ -89,7 +89,16 @@ def draw_bar_plot(data, plot_attributes, xlabel=None, y_limit=None):
     num_bars = len(data.items()) - 1
     if is_grouped:
         x_ticks = dict()
-        x_tick_info = dict([(group, [groups.index(group), 0]) for group in groups])
+
+        # fiddly stuff to get groups of uneven sizes to have even space betweem them
```
Comment (karlowich) [inline cijoe/scripts/latency_plotter.py:93]:
```suggestion
        # fiddly stuff to get groups of uneven sizes to have even space between them
```

**Review by karlowich — CHANGES_REQUESTED** [review]
Looks good.
I just have a few nitpicks

File: cijoe/scripts/bench_plotter.py (hunk 1 above)
Comment (NaddiNadja) [reply cijoe/scripts/bench_plotter.py]: I have set both to None, and updated the `latency_plotter` to have same format, such that when importing additional scripts in CIJOE is supported, we can easily extract this function into a shared imported script.

**Review by safl — APPROVED** [review]
lgtm

**Review by karlowich — APPROVED** [review]

**Pull request merged**